src/satisfiers/route.ts

```diff
@@ -0,0 +1,19 @@
+import { Satisfier, Caveat } from 'lsat-js'
+import { Request } from 'express'
+const { MASTER_ROUTE } = process.env
+
+/**
+ * @description A satisfier for validating caveats based on the original route
+ * used in the exported boltwallConfig ROUTE_CAVEAT_CONFIGS
+ * @type Satisfier
+ */
+const routeSatisfier: Satisfier = {
+  condition: 'route',
+  satisfyFinal: (caveat: Caveat, req: Request) => {
+    const path = req.path
+    if (caveat.condition === 'route' && (caveat.value === path || caveat.value === MASTER_ROUTE)) return true
```

Comment:
I'm thinking that an alternative way to do the MASTER_ROUTE such that you don't have to add global values to the process environment, and should simplify testing such that you don't have to add an environment variable to get tests to work, is to extend the `BoltwallConfig` (which gets attached to the request object so that other middlewares can have access to it, see [this](https://stackoverflow.com/questions/37377731/extend-express-request-object-using-typescript) SO question for ideas about extending the `Request` object).

This makes testing much easier because rather than adding global variables you can just mock different request objects/boltwall configs and that way also test different values for this configuration.

Reply:
Awesome. This is a much better way, thank you. I didn't see that boltwallConfig was getting added to the request. I pushed changes to fix this. Thanks!